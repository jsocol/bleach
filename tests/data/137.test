<div id="65"><svg onload="javascript:alert(65)" xmlns="http://www.w3.org/2000/svg"></svg>//["'`-->]]>]</div><div id="66"><?xml version="1.0"?>
