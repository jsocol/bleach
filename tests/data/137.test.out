<div id="65"><svg xmlns="http://www.w3.org/2000/svg"></svg>//["'`--&gt;]]&gt;]</div><div id="66"></div>
